### PR TITLE
feat: add KitchenSink Stories for all components

### PR DESCRIPTION
**Pull request by kamleshchandnani**
This PR adds `KitchenSink/<ComponentName>` stories for all the components and enables snapshots only on KitchenSink

### CODE REVIEW

File: packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
```
Comment (anuraghazra) [inline packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx:1]: whats with this `_` convention? is this needed for differentiation?

File: packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import * as accordionStories from './Accordion.stories';
+import { Box } from '~components/Box';
+import { Heading } from '~components/Typography';
+
+const allStories = Object.values(composeStories(accordionStories));
+
+export const Accordion = (): JSX.Element => {
```
Comment (anuraghazra) [inline packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx:8]: in react-native do these kitchen sinks exists? If so i hope they won't pollute the sidebar unnecessarily 😢

**Review by anuraghazra — APPROVED** [review]
Looks good.

File: packages/blade/src/components/SkipNav/_KitchenSink.SkipNav.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
```
Comment (anuraghazra) [inline packages/blade/src/components/SkipNav/_KitchenSink.SkipNav.stories.tsx:1]: We can remove this one, unless the nav link is focused it won't show up and anyway it's just a link only.

File: packages/blade/src/components/Typography/Heading/_KitchenSink.Heading.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
+import * as headingStories from './Heading.stories';
+import { Box } from '~components/Box';
+import { Heading } from '~components/Typography';
+
+const allStories = Object.values(composeStories(headingStories));
+
+export const Heading = (): JSX.Element => {
```
Comment (snitin315) [inline packages/blade/src/components/Typography/Heading/_KitchenSink.Heading.stories.tsx:8]: It's conflicting with the imported `Heading` component.

```suggestion
export const HeadingStory = (): JSX.Element => {
```

File: packages/blade/src/components/Typography/Heading/_KitchenSink.Heading.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+          </>
+        );
+      })}
+    </Box>
+  );
+};
+
+export default {
+  title: 'Components/KitchenSink/Heading',
+  component: Heading,
```
Comment (snitin315) [inline packages/blade/src/components/Typography/Heading/_KitchenSink.Heading.stories.tsx:25]:
```suggestion
  component: HeadingStory,
```

File: packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx [hunk 6]
```diff
@@ -216,7 +216,7 @@ export default {
 } as Meta<StoryControlProps>;
 
 const BottomSheetTemplate: ComponentStory<typeof BottomSheetComponent> = ({ ...args }) => {
-  const [isOpen, setIsOpen] = React.useState(false);
+  const [isOpen, setIsOpen] = React.useState(true);
```
Comment (snitin315) [inline packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx:219]: Is it open by default now? It seems not related.

File: packages/blade/src/components/VisuallyHidden/_KitchenSink.VisuallyHidden.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,31 @@
+import { composeStories } from '@storybook/react';
```
Comment (snitin315) [inline packages/blade/src/components/VisuallyHidden/_KitchenSink.VisuallyHidden.stories.tsx:1]: I believe we don't need a kitchen sink story for this as this component doesn't show up in UI.

File: packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx (hunk 1 above)
Comment (kamleshchandnani) [reply packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx:1]: yes plus this moves the story at the top otherwise alphabetically sort hota hai to it looks ugly on sidebar amidst your main files

File: packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx (hunk 2 above)
Comment (kamleshchandnani) [reply packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx:8]: but that shouldn't matter na because Kitchen Sink is a directory and it will come at the last

File: packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx (hunk 6 above)
Comment (kamleshchandnani) [reply packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx:219]: it is not open by default but i made this change because otherwise snapshot me bottomsheet won't be captured as its closed and we'll get snapshot of only "Open" button. we do this for Modal as well

Comment (snitin315) [reply packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx:219]: Oh! Makes sense.

Comment (anuraghazra) [reply packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx:219]: Makes sense, although @kamleshchandnani is it possible to detect if we are running on chromatic snapshot testing and only then make it true? Because now whenever you open the BottomSheet ka docs this bottomsheet will open up and interrupt the consumer.

File: packages/blade/src/components/VisuallyHidden/_KitchenSink.VisuallyHidden.stories.tsx (hunk 7 above)
Comment (anuraghazra) [reply packages/blade/src/components/VisuallyHidden/_KitchenSink.VisuallyHidden.stories.tsx:1]: Rehne dete hain, what if suddenly <VisuallyHidden \/> becomes <OhShitItsNotHiddenAnymore \/>

File: packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx (hunk 2 above)
Comment (anuraghazra) [reply packages/blade/src/components/Accordion/_KitchenSink.Accordion.stories.tsx:8]: Oh accha 6.5 mein we have nice directories in RN also, okay.

File: packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx (hunk 6 above)
Comment (kamleshchandnani) [reply packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx:219]: I can yes. but made it same as Modal. can do it for both then

File: packages/blade/src/components/VisuallyHidden/_KitchenSink.VisuallyHidden.stories.tsx (hunk 7 above)
Comment (kamleshchandnani) [reply packages/blade/src/components/VisuallyHidden/_KitchenSink.VisuallyHidden.stories.tsx:1]: yes parity

File: packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx (hunk 6 above)
Comment (kamleshchandnani) [reply packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx]: done [here](https://github.com/razorpay/blade/pull/1542/commits/678b74bdf82bb668737c16207eb6ed0519621b86)

**Review by snitin315 — APPROVED** [review]
:shipit:
